packages/truffle-workflow-compile/README.md

```diff
@@ -20,13 +20,13 @@ const config = {
 
 // compiles contracts found in contracts_directory,
 // saves them in contracts_build_directory
-Contracts.compile(config, () => {
-  console.log("Compilation complete!");
-});
+Contracts.compile(config)
+  .then(() => console.log("Compilation complete!"))
+  .catch(e => console.error(e))
 
-// alternative making use of the expected callback
+// callback alternative
```

Comment:
I wonder if going forward we want to even deprecate the callback interface. I'm not sure.

Reply:
Yeah, not sure if I should leave that example in for now or strip it out.

Reply:
Toss it. No one wants to hear about callbacks.